pyproject.toml

```diff
@@ -17,20 +18,32 @@ packages = [
     { include = "mkdocs_drawio/plugin.py" },
 ]
 
+[tool.poetry.urls]
+"Documentation" = "https://github.com/tuunit/mkdocs-drawio/"
+"Source" = "https://github.com/tuunit/mkdocs-drawio/"
+"Tracker" = "https://github.com/tuunit/mkdocs-drawio/issues"
+
 [tool.poetry.dependencies]
-python = "^3.7.0"
-requests = ">=2.0"
-Jinja2 = ">=3.0"
+python = ">=3.8.0,<4.0"
 beautifulsoup4 = ">=4.0"
 lxml = ">=4.0"
 mkdocs = ">=1.3"
 
-[tool.poetry.dev-dependencies]
+[tool.poetry.group.dev.dependencies]
+poetry = "^2.0"
+python = ">=3.9.0,<4.0"
 black = ">=24.0"
+ruff = "^0.9.2"
+mkdocs-print-site-plugin = "^2.6.0"
+mkdocs-material = "^9.5.50"
+mkdocs-glightbox = "^0.4.0"
```

Comment:
I don't want to force people to install dependencies they might not need

Reply:
That's why those packages are in the dev-dependencies. In the CI, the build command would only use `poetry install --no-dev`. This is what `poetry build` do.